Setup.lhs

```diff
@@ -72,33 +53,10 @@ generateBuildModule verbosity pkg lbi = do
         , ""
         , "compiler :: String"
         , "compiler = " ++ (show $ showCompilerId $ compiler lbi)
-        , ""
-        , "depsFlag :: String"
-#if MIN_VERSION_Cabal(1,24,0)
-        , "depsFlag = " ++ show "-package-id="
-#else
-        , "depsFlag = " ++ show "-package="
-#endif
```

Comment:
`package-id` is more precise (you can have single version of a package installed multiple times, which happens with new-build more often than you might think at first). Before realising that `display` did *the right* thing, even `InstalledPackageId` representation changed in between, I didn't know how to extract the package-id string without CPP magic (which, unfortunately, wasn't reliable)